guides/common/modules/proc_configuring-provisioning-callback-for-a-host.adoc

```diff
@@ -47,8 +47,9 @@ For more information about job templates, see http://docs.ansible.com/automation
 |====
 |Name |Value |Description
 |`ansible_tower_provisioning` |true |Enables Provisioning Callback.
-|`ansible_tower_fqdn` |_controller.example.com_ |The fully qualified domain name (FQDN) of your {awx}.
-Do not add `https` because this is appended by {Project}.
+|`ansible_tower_api_url` |_https://controller.example.com/api/controller/v2_ | Defines the URL of your {awx}, including the required API path.
+For older versions of {awx}, use `/api/v2` instead of `/api/controller/v2`.
```

Comment:
```suggestion
For {awx} versions earlier than X.Y, use `/api/v2` instead of `/api/controller/v2`.
```
Similar to the RN PR, please specify the version if possible.

Reply:
@maximiliankolb @aneta-petrova can we mark it as resolved?

Reply:
@maximiliankolb In https://github.com/theforeman/foreman-documentation/pull/3720, the line got merged in its original state (`older {awx} version`). Does it mean we can merge this PR as well?

Reply:
Yes, I am OK as is.